winrt/lib/text/CanvasFontFace.h

```diff
@@ -122,6 +122,24 @@ namespace ABI { namespace Microsoft { namespace Graphics { namespace Canvas { na
             uint32_t* outputCount,
             int** outputElements) override;
 
+        IFACEMETHOD(GetGlyphMetrics)(
+          uint32_t inputCount,
+          uint16_t* inputElements,
+          boolean isSideways,
+          uint32_t* outputCount,
+          CanvasGlyphMetrics** outputElements) override;
+
+        IFACEMETHOD(GetGdiCompatibleGlyphMetrics)(
+          float emSize,
+          float pixelsPerDip,
+          Numerics::Matrix3x2 transform,
+          boolean useGdiNatural,
+          uint32_t inputCount,
+          UINT16* inputElements,
```

Comment:
We try and use the C++ standard typenames wherever possible - uint16_t in this case.